pycqed/analysis/measurement_analysis.py

```diff
@@ -60,19 +60,34 @@ def __init__(self, TwoD=False, folder=None, auto=True,
         self.fit_results = []
         self.cmap_chosen = cmap_chosen
         self.no_of_columns = no_of_columns
-        self.dpi = 600                  #dpi for plots
-        self.axes_line_width=0.5        #lw of axes and text boxes
-        self.font_size = 11             #font sizes
-        self.tick_length = 4            #tick lengths
-        self.tick_width = 0.5           #tick line widths
-        self.marker_size=3              #marker size for data points
-        self.line_width=2               #line widths connecting data points
-        self.marker_size_special=8      #marker size for special points like
-                                        #peak freq., Rabi pi and pi/2 amplitudes
+        self.dpi = kw.get('dpi', 300)  # dpi for plots
+        # 300 dpi is a good balance between resolution and speed of analysis
```

Comment:
Writing these parameters with kw instead of hardcoding them is fixed in PR #347.

Reply:
@Vintrix I looked at your PR, I think putting them in a separate method is nicer (as in your PR).  I suggest we include that improvement when going over your PR (probably somewhere next week).